algorithm_rgb.py

```diff
@@ -6,20 +6,20 @@
 
 # Definitions
 # Please replace these definitions' values with the correct ones
-VERSION = '1.0'
+VERSION = '2.0'
```

Comment:
This should be completely filled out now since it's an implementation of an algorithm

Reply:
How should I update there version. I've been looking through AgPipeline for examples but haven't found one of an algorithm implementation yet.